marketplaceItems/sitPoint/appResources/json/opaqueSitCube.json

```diff
@@ -0,0 +1,67 @@
+{
+  "DataVersion": 0,
+  "Entities": [
+    {
+    "id": "{eedbf4a1-a667-42c4-b287-c5fa14001571}",
+    "type": "Box",
+    "created": "2019-01-16T23:05:56Z",
+    "lastEdited": 1547680135056164,
+    "lastEditedBy": "{77356988-272b-49e6-b445-4768947ca89f}",
+    "dimensions": {
+        "x": 0.20000000298023224,
+        "y": 0.20000000298023224,
+        "z": 0.20000000298023224
+    },
+    "rotation": {
+        "x": -0.0000152587890625,
+        "y": -0.0000152587890625,
+        "z": -0.0000152587890625,
+        "w": 1
+    },
+    "damping": 0,
+    "script": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/marketPlaceItems/sitPoint/2019-01-16_11-07-10/appResources/appData/sitClient.js",
+    "scriptTimestamp": 1547680112787,
+    "serverScripts": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/marketPlaceItems/sitPoint/2019-01-16_11-07-10/appResources/appData/sitServer.js",
```

Comment:
Please replace this link with relative links for GitHub and make a note in the README that you have to change the script for one's own usage.